latex/ledger-spec.tex

```diff
@@ -604,6 +884,36 @@ \subsection{Witnesses}
   \label{fig:derived-defs:utxow}
 \end{figure}
 
+
+Note that the UTxO transitions with and without witnesses have the same type
+(see Figure~\ref{fig:ts-types:utxo} and Figure~\ref{fig:ts-types:utxow}).
+The witnessed transition rule, in fact, defines the same UTxO update as the
+non-witnessed rule. Essentially, the witnessed rule says we may only apply the
+the non-witnessed rule whenever the following
+additional precondition is satisfied
+(stated in Rule~\ref{eq:utxo-witness-inductive} in
+Figure~\ref{fig:rules:utxow}):
+
+\begin{itemize}
+ \item For each input $\var{i}$ of the signal transaction $\var{tx}$,
+ there exists a verifiable witness (there could be more than one valid
+ key-signature pair serving as a witness) such the hash of the spending key
+ in the address corresponding to the output of $\var{i}$ in the current UTxO
+ is the same as the hash of the verification key of the witness.
+\end{itemize}
+
+The purpose of witnessing is to provide replay protection. Due to the nature
```

Comment:
witnessing is about more than just replay attacks, it's about knowing that the action is authorized.  ie that the holder of the signing key intended this action. but indeed the certificates piggyback on the UTxO's inherent replay protection.

Reply:
Right!